src/lib/background/session/storage.js

```diff
@@ -39,7 +39,7 @@ function addAttention(tabId, url, addedAttention, isManual)
 
         if (addedAttention >= getRemainingAttention(entity, oldAttention))
         {
-          yield flattrManager.submit({
+          flattrManager.submit({
```

Comment:
> At least I could imagine that this leads to inconsistencies in the UI (e.g. attention bar filling up but flattr count not increasing).

At the moment if there is an error then nothing in the UI changes, is that better?  I don't think it is, I think it's worse because we're just hiding errors from the user.

Also we reduce the chances of an error here because we make sure that only one save is performed at a time, and so we don't depend on transactions (which can cause errors).

Finally we at least have the chance to correct the error here, with logic that can detect error states like the one that you pointed out above (Pretty sure I didn't do this here yet, but I think that would be a good follow up issue).